commands/access/remove.js

```diff
@@ -1,8 +1,28 @@
 'use strict';
 
-var Heroku      = require('heroku-client');
-var co          = require('co');
-var heroku;
+let cli           = require('heroku-cli-util');
+let Utils         = require('../../lib/utils');
+let co            = require('co');
+
+function* run(context, heroku) {
+  let appName = context.app;
+  let appInfo = yield heroku.apps(appName).info();
+  let request;
+
+  if (Utils.isOrgApp(appInfo.owner.email)) {
+    request = heroku.request({
+      method: 'DELETE',
+      path: `/organizations/apps/${appName}/collaborators/${context.args.email}`,
```

Comment:
That's what dashboard is currently doing depending on whether is a personal
or an org app.

I assumed DELETE /apps/:app/collaborators/:user would work for org apps as
well, but that's not the case. It returns a 404 in those.
On Tue, Feb 23, 2016 at 12:00 PM naaman <notifications@github.com> wrote:

> In commands/access/remove.js
> <https://github.com/heroku/heroku-orgs/pull/22#discussion_r53840413>:
>
> > -var Heroku      = require('heroku-client');
> > -var co          = require('co');
> > -var heroku;
> > +let cli           = require('heroku-cli-util');
> > +let Utils         = require('../../lib/utils');
> > +let co            = require('co');
> > +
> > +function* run(context, heroku) {
> > +  let appName = context.app;
> > +  let appInfo = yield heroku.apps(appName).info();
> > +  let request;
> > +
> > +  if (Utils.isOrgApp(appInfo.owner.email)) {
> > +    request = heroku.request({
> > +      method: 'DELETE',
> > +      path: `/organizations/apps/${appName}/collaborators/${context.args.email}`,
>
> AFAIK, deleting a collaborator is the same. The difference between the org
> collab and regular collab endpoints are updating user privileges.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/heroku/heroku-orgs/pull/22/files#r53840413>.
>